core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/iterator/GroupIterator.java

```diff
@@ -474,10 +477,14 @@ public MinAggregate(Min operator) {
 		@Override
 		public void processAggregate(BindingSet s) throws QueryEvaluationException {
 			Value v = evaluate(s);
+			boolean strict = false;
+			if (strategy instanceof StrictEvaluationStrategy) {
+				strict = true;
+			}
```

Comment:
This won't quite work. Pretty much _every_ EvaluationStrategy implementation in RDF4J is a subclass of `StrictEvaluationStrategy`. Even `ExtendedEvaluationStrategy`. 

I think you need to do this the other way around to make it work:

                        boolean strict = true;
			if (strategy instanceof ExtendedEvaluationStrategy) {
				strict = false;
			}

Reply:
Okay I made the check to `ExtendedEvaluationStrategy`.

Reply:
But It is still failing on some test cases in build.